lochness/rpms/__init__.py

```diff
@@ -188,9 +202,10 @@ def initialize_metadata(Lochness: 'Lochness object',
                                         df_measure[rpms_id_colname]
 
             # if mindlamp_id exists in the rpms table
-            if 'mindlamp_id' in df_measure:
-                subject_dict['mindlamp'] = f'mindlamp.{study_name}:' \
-                        + df_measure[f'mindlamp_id']
+            if 'chrdbb_lamp_id' in df_measure:
+                if not pd.isna(df_measure[f'chrdbb_lamp_id']):
+                    subject_dict['Mindlamp'] = f'mindlamp.{study_name}:' \
+                            + df_measure[f'chrdbb_lamp_id']
```

Comment:
Can you confirm that this block is unrelated to the feature I requested?

Reply:
Correct. This is resolving `mindlamp` run sheet name change issue.